backend/scoring/scoring.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+)
+
+type Score struct {
```

Comment:
These structus overlap with the funcitonality provided in [game.go](https://github.com/jak103/powerscore/blob/main/backend/trimmed.txt), specifically the TeamData struct.